decidim-api/spec/types/mutation_type_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require "spec_helper"
+require "decidim/api/test/type_context"
+
+module Decidim
+  module Api
+    describe MutationType do
+      include_context "graphql type"
+
+      describe "addComment" do
+        let!(:participatory_process) { create(:participatory_process, organization: current_organization) }
+
+        let(:query) { 
+          "{ addComment(commentableId: \"#{participatory_process.id}\", commentableType: \"Decidim::ParticipatoryProcess\", body: \"This is a new comment\") { id, body } }" 
```

Comment:
It feels weird to me that the frontend knows internals of the backend (ie the name of the class of the participatory process). The `CommentsHelper` sets automatically this field (`resource.class.name`), but it still feels weird to me that the frontend knows these things. If we want to make a mobile app that is not linked with the Rails backend, so that it can't set `resource.class.name`, then the mobile app needs to know the name of the Rails class.